packages/react-ui-ag/src/Modals/MoreOptionsModal.js

```diff
@@ -0,0 +1,163 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import themed from 'react-themed'
+import autobind from 'autobind-decorator'
+import classnames from 'classnames'
+import { Modal, Button, Text } from '@rentpath/react-ui-core'
+
+@themed(/^MoreOptionsModal/, { pure: true })
+
+export default class MoreOptionsModal extends PureComponent {
+  static propTypes = {
+    isOpen: PropTypes.bool,
+    theme: PropTypes.object,
+    moreOptionsForm: PropTypes.node,
+    total: PropTypes.number,
+    filterTotal: PropTypes.number,
+    closeModal: PropTypes.func,
+    onSubmit: PropTypes.func,
+    clearFilters: PropTypes.func,
+    restoreFilters: PropTypes.func,
+    className: PropTypes.string,
+  }
+
+  static defaultProps = {
+    theme: {},
+    clearFilters: () => {},
+    onSubmit: () => {},
+    closeModal: () => {},
+    restoreFilters: () => {},
+  }
+
+  getSubmitButton() {
+    const { filterTotal, total } = this.props
+
+    const count = (filterTotal !== null) ? filterTotal : total
+    let propertyCount = 'Show 0 Properties'
+
+    if (count > 1) {
+      propertyCount = `Show ${count} Properties`
+    } else if (count === 1) {
+      propertyCount = 'Show 1 Property'
+    }
+
+    return propertyCount
+  }
+
+  getPropertyCount() {
+    const { filterTotal, total } = this.props
+    const count = (filterTotal !== null) ? filterTotal : total
+    let propertyCount = '0 Properties Found'
+
+    if (count > 1) {
+      propertyCount = `${count} Properties Found`
+    } else if (count === 1) {
+      propertyCount = '1 Property Found'
+    }
+
+    return propertyCount
+  }
+
+  @autobind
+  handleSubmit() {
+    const {
+      onSubmit,
+      closeModal,
+    } = this.props
+
+    if (onSubmit) {
+      onSubmit()
+    }
+    closeModal()
+  }
+
+  @autobind
+  handleCancel() {
+    const {
+      closeModal,
+      restoreFilters,
+    } = this.props
+    restoreFilters()
+    closeModal()
+  }
+
+  renderHeader() {
+    const {
+      theme,
+    } = this.props
+
+    const propertyCount = this.getPropertyCount()
+
+    return (
+      <div className={theme.MoreOptionsModal_Header}>
+        <Text>
+          {propertyCount}
+        </Text>
+        <Button
+          onClick={this.handleCancel}
+        >
+          Close
+        </Button>
+      </div>
+    )
+  }
+
+  renderFooter() {
+    const {
+      theme,
+    } = this.props
+
+    const submitButton = this.getSubmitButton()
+
+    return (
+      <div className={theme.MoreOptionsModal_Footer}>
+        <Button
+          className={theme.MoreOptionsModal_ResetAll}
+          onClick={this.props.clearFilters}
+        >
```

Comment:
I don't think I could write a more knit picky comment, but the `onClick` could be ran through the props destructure like `theme`. Other than that everything looks great.

Reply:
Fixed